src/com/hw/airport/service/FlightSvcImpl.java

```diff
@@ -3,11 +3,28 @@
 import com.hw.airport.exception.HWAirportException;
 import com.hw.airport.exception.MissingFlightException;
 import com.hw.airport.model.AppData;
+import com.hw.airport.model.Booking;
 import com.hw.airport.model.Flight;
 
 public class FlightSvcImpl implements FlightSvc {
 	AppData appData = AppData.getInstance();
+	
+	BaggageSvc baggageService = new BaggageSvcImpl();
+	// 1. check if flight is overloaded for volume / weight / passenger count
 
+	
+	@Override
+	public int getPassengerCountForFlight(String flightCode) throws HWAirportException {
+		int currentCheckedInPsngrCount = 0;
+		for(Booking booking : appData.getBookingList().values()) {
+			boolean isBookingOnFlightAndCheckedIn = (booking.getFlightCode().equals(flightCode)) && booking.isCheckedIn();
+			if(isBookingOnFlightAndCheckedIn) {
```

Comment:
To simplify the code, no need to define a variable to hold the Boolean value. you can just use it directly in the "if statement " condition clause